### ISSUE TITLE
Combine UNet with Power Perceiver

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/power_perceiver#63]
- [x] Run a Power Perceiver with the same params, but with "cheating" imagery.
- [x] Don't train the U-Net (`.detach()`)
- [x] #64
- [x] #65
- [ ] Check that the `einops.reduce` is doing what we expect in `HRVSatelliteProcessor`
- [x] Train the U-Net. Include a satellite loss. Although need to only use the future images that are in the training batch.
- [x] LR scheduling for Power Perceiver.
- [x] Use 128x128 imagery input to the satellite predictor
- [ ] NWP
